app/src/main/res/drawable/quiet_hour_icon.xml

```diff
@@ -0,0 +1,27 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
which icon is this?

Reply:
@rob729  i downloaded it from FlatIcon.com, it is the same icon which is already present in the project... but old icon was not looking sharp, So, i downloaded the same vector image icon.